### ISSUE TITLE
rollup build is complaining about circular dependencies in react-bootstrap-typeahead

### ISSUE THREAD

**Issue by SerdarSanri** [ericgio/react-bootstrap-typeahead#424]
### Version
4.1.0

### Steps to reproduce
add module to your project, and run with `rollup -c`

### Expected Behavior
Bundled application should build without warnings

### Actual Behavior
rollup build sends two warnings that react-bootstrap-typeahead is using circular dependency which causes application to become unresponsive
```
(!) Circular dependency: node_modules\react-bootstrap-typeahead\lib\utils\index.js -> node_modules\react-bootstrap-typeahead\lib\utils\addCustomOption.js -> node_modules\react-bootstrap-typeahead\lib\utils\index.js
(!) Circular dependency: node_modules\react-bootstrap-typeahead\lib\utils\index.js -> node_modules\react-bootstrap-typeahead\lib\utils\addCustomOption.js ->  commonjs-proxy:C:\project\node_modules\react-bootstrap-typeahead\lib\utils\index.js -> node_modules\react-bootstrap-typeahead\lib\utils\index.js
```

**Comment by ericgio** [comment]
Thanks for the report! Confirmed the problem.

**Comment by ericgio** [comment]
Fixed in v3.3.5 + added a webpack plugin to catch future occurrences.
